app/views/teacher_training_adviser/steps/_overseas_time_zone.html.erb

```diff
@@ -1,7 +1,8 @@
 <%= f.govuk_fieldset legend: { text: 'You told us you have an equivalent degree and live overseas' } do %>
 
-<p>We will need to call you to check your degree. You must have the details of your overseas qualifications when we contact you.</p>
-<p>In the meantime, you can call us on Freephone <%= link_to("0800 389 2500", "tel://08003892500") %> between 8.30am and 5.30pm Monday to Friday.</p>
+<p>We will need to call you to check your degree before you can get an adviser.</p>
+
+<p>We will call you between 8.30am and 5.30pm (UK time).</p>
```

Comment:
just to check @Tomcowan1, what's the logic for removing option to call themselves in meantime?

Reply:
Good question @MylesJarvis. When I was reading through the original I thought it was a bit confusing - we're saying we'll call people, but then telling them to call us at the same time. Much easier to have one or the other. 

Plus, we used to say "have the details ready when we contact you" which was a bit odd - we shouldn't expect people to carry them around until we call.

Reply:
Hi @MylesJarvis I've edited this one and am checking it again with Sarah BH. The other two changes (confirmation page / telephone number) are good to go, if they can be deployed separately?